index.d.ts

```diff
@@ -0,0 +1,21 @@
+interface BaseSignOptions {
+  app: string;
+  identity?: string;
+  platform?: string;
+  keychain?: string;
+}
+
+interface SignOptions extends BaseSignOptions {
+  binaries?: string[];
+  entitlements?: string;
+  'entitlements-inherit'?: string;
+}
+
+export function sign(opts: SignOptions, callback: (error: Error) => void): void
```

Comment:
@develar It seems that GitHub in some ways acknowledges the semicolon... I'm not sure. Only with the semicolon added could the next line (interface) be correctly colored. Shall we follow the strict rule by adding one in the end? 😕